communication/nats_discovery/address.go

```diff
@@ -0,0 +1,77 @@
+package nats_discovery
+
+import (
+	"fmt"
+	dto_discovery "github.com/mysterium/node/service_discovery/dto"
+	"github.com/nats-io/go-nats"
+)
+
+func NewAddress(server string, port int, topic string) *NatsAddress {
+	return &NatsAddress{
+		servers: []string{
+			fmt.Sprintf("nats://%s:%d", server, port),
+		},
+		topic: topic,
+	}
+}
+
+func NewAddressForIdentity(identity dto_discovery.Identity) *NatsAddress {
+	return NewAddress("127.0.0.1", 4222, string(identity))
```

Comment:
Is it ok to hardcode "127.0.0.1" + port here?

Reply:
Maybe worth changing into a variable we could modify during compilation?

Reply:
It's the same how it was before, we expect queue server to run next to node&client.
We have a ticket to make it injectable